# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 1e4f8d9d96ce
# Full Hash: 1e4f8d9d96ce1ccb79c154416b119907f9479294
# Author: Mark Striemer <mstriemer@mozilla.com>
# Date: 2020-06-04 09:29:07
# Regressor Bug: 1632277
# File Overlap Count: 1
# Description:
#   Bug 1632277 - Part 1: Launch PDFs in app mode when default r=jaws,tkikuchi
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D73774
# ==============================================================================

diff -r 04d728d14947 -r 1e4f8d9d96ce modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Wed Jun 03 23:41:48 2020 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Wed Jun 03 18:21:32 2020 +0000
@@ -984,6 +984,12 @@
   value: true
   mirror: always
 
+# Open PDFs in Edge with the --app flag if it is the default.
+- name: browser.pdf.launchDefaultEdgeAsApp
+  type: bool
+  value: true
+  mirror: always
+
 # Force usage of in-memory (rather than file on disk) media cache for video streaming when private browsing
 - name: browser.privatebrowsing.forceMediaMemoryCache
   type: bool